### DISCUSSION TITLE
Velero CRD versions

### DISCUSSION THREAD

**Discussion by niranjan-gavade** [vmware-tanzu/velero#3458] [Community support Q&A] [answered]
Can we assume that all Velero CRD's version will be same? i.e. Is it possible that Backup resource version is X & VolumeSnapshotLocation version is Y?

**Comment by ashish-amarnath** [comment, marked as answer]
Velero will always try to maintain backward compatibility with its api objects. All breaking changes will be announced giving sufficient notice for adoption.

I hope this answers your question.
